pocs/camera/camera.py

```diff
@@ -311,6 +311,22 @@ def set_property(self, prop, val):
         # Forces the command to wait
         self.get_command_result()
 
+    def set_property_values(self, prop_vals):
+        """ Sets a number of properties all at once
+
+        Args:
+            prop_vals (dict): A dict with keys corresponding to the property to
+            be set and values corresponding to the index option
+        """
+        set_cmd = list()
+        for prop, val in prop_vals.items():
+            set_cmd.append('--set-config-index {}={}'.format(prop, val))
```

Comment:
I note that the set_property function has the flag (--set-config) in a separate list entry from the prop=val entry.

Reply:
These all get passed to a `subprocess` which accepts a list of options for the command.  It could probably be set up either way, it would just require to `append` statements here. The original `set_property` only has two options so it just constructs it as a list to begin with and avoids the `append`. Would be worth investigating if there is any practical difference in the end.